notebooks/leb_salt_evaluation.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    auth_token = "xxxxx"
</pre><p>maybe test if <code>os.environ["HF_TOKEN"]&nbsp;</code> has been set, and if not then use <code>getpass</code> to define the auth token? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/leb/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>